### PR TITLE
[569451] Migration of model to support configurable Security Needs

### PR DESCRIPTION
**Pull request by georgiana-ecobici**
Change-Id: If4d80943ee7dd8f6decc97979fd326a37fd34992
Signed-off-by: Georgiana-Elena ECOBICI <georgiana-elena.ecobici@thalesgroup.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/contribution/migration/CybersecurityMigrationContribution.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import org.polarsys.capella.cybersecurity.model.activator.CybersecurityModelActivator;
+
+public class CybersecurityMigrationContribution extends AbstractMigrationContribution {
+  Map<String, String> savedValues = new HashMap<>();
+  Project project;
+
+  @Override
+  public void unaryMigrationExecute(EObject currentElement, MigrationContext context) {
+    super.unaryMigrationExecute(currentElement, context);
+    if (currentElement instanceof Project) {
```
Comment (minhtutonthat) [inline plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/contribution/migration/CybersecurityMigrationContribution.java:36]: Does it mean that you always create this configuration folder even though the viewpoint is not activated on the project ? A check should be done here.

**Review by minhtutonthat — CHANGES_REQUESTED** [review]

**Comment by minhtutonthat** [comment]
And is it normal that there are always 6 KO tests?

File: plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/contribution/migration/CybersecurityMigrationContribution.java (hunk 1 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/contribution/migration/CybersecurityMigrationContribution.java:36]: Ok.

Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/contribution/migration/CybersecurityMigrationContribution.java:36]: I have checked that there are values that need migration.

**Review by minhtutonthat — APPROVED** [review]

**Pull request merged**